docs/api/modules/App-Configuration.md

```diff
@@ -454,6 +456,25 @@ The `eventLoopDelayThreshold` directive accepts a number representing the thresh
 * [Frank Lloyd Root's `appConfig`](../../../prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/config.js)
 * Library: [Opossum](https://nodeshift.dev/opossum/)
 
+## `errorPageUrl`
+
+**Module Type**
+
+- ✅ Root Module
+- 🚫 Child Module
+
+**Shape**
+
+```js
+if (!global.BROWSER) {
+  Module.appConfig = {
+    errorPageUrl: String,
+  };
+}
+```
+
+The `errorPageUrl` directive is useful for supplying a URL to a custom error page. This URL needs to return a `Content-Type` of `text/html` and a `Content-Length` of less than `50000`. The URL will get called when your root module is loaded and is rendered if and when an error occurs.
```

Comment:
where did the `< 50000` limit come from ?

Reply:
@JAdshead it was a limit that @mtomcal and I came up with randomly I will admit. We wanted to discuss during the code review.

Reply:
@JAdshead I changed this to be the default size of our modules